### PR TITLE
For YOLO-v3 model conversion info expanded

### PR DESCRIPTION
**Pull request by eizamaliev**

### CODE REVIEW

File: models/public/yolo-v3-tf/yolo-v3-tf.md [hunk 1]
```diff
@@ -7,7 +7,17 @@ YOLO v3 is a real-time object detection model implemented with Keras\* from this
 ## Conversion
 
 1. Download or clone the official [repository](https://github.com/david8862/keras-YOLOv3-model-set) (tested on `ffede5` commit).
-2. Follow the instructions in README.md file in that repository to get original model and convert it to Keras\* format.
+2. Follow the instructions in README.md file in that repository to get original model (named `yolov3` in repository) and convert it to Keras\* format **OR** follow these steps:
```
Comment (IRDonch) [inline models/public/yolo-v3-tf/yolo-v3-tf.md]: How can there be alternatives? You could have only done one of these when you converted the model.

Comment (LeonidBeynenson) [reply models/public/yolo-v3-tf/yolo-v3-tf.md]: I'd propose to rewrite that as follows:
```suggestion
2. Use the folllowing commands to get original model (named `yolov3` in repository) and convert it to Keras\* format (see details in the [README.md](https://github.com/david8862/keras-YOLOv3-model-set/README.md)  file in the official repository):
```

Comment (eizamaliev) [reply models/public/yolo-v3-tf/yolo-v3-tf.md]: It is just some main items from repo's README.

Comment (IRDonch) [reply models/public/yolo-v3-tf/yolo-v3-tf.md]: > It is just some main items from repo's README.

So why replicate them?

Comment (IRDonch) [reply models/public/yolo-v3-tf/yolo-v3-tf.md]: Never mind, I didn't see your changes.

Comment (IRDonch) [reply models/public/yolo-v3-tf/yolo-v3-tf.md]: Wait, I'm mistaken. I thought you removed the expanded text. In that case, the question remains.

Comment (eizamaliev) [reply models/public/yolo-v3-tf/yolo-v3-tf.md]: Because there are several models in the repo, so we just specifying model.

**Comment by LeonidBeynenson** [comment]
2 @eizamaliev : I'm sorry, but the phrase 
```
Follow the instructions in README.md file in that repository to get original model (named `yolov3` in repository) and convert it to Keras\* format **OR** follow these steps
```
-- is not correct, since user will think that there are 2 **different** options: either follow instructions in README.md, **OR** follow the steps, and the options will work in **different** way.

As I stated above, I'd propose to rewrite that as follows:
```
2. Use the folllowing commands to get original model (named `yolov3` in repository) 
   and convert it to Keras\* format (see details in the [README.md](https://github.com/david8862/keras-YOLOv3-model-set/README.md)  file in the official repository):
```

**Comment by IRDonch** [comment]
> since user will think that there are 2 different options

Can confirm, this is exactly what I thought. :-)

**Comment by eizamaliev** [comment]
@IRDonch @LeonidBeynenson is it ok now?

**Pull request merged**